doc/introduction/compiling_workflows.rst

```diff
@@ -7,12 +7,15 @@ Compiling workflows
 ===================
 
 In addition to :doc:`quantum circuit transformations </introduction/compiling_circuits>`, PennyLane also supports full
-hybrid just-in-time (JIT) compilation via the :func:`~.qjit` decorator and
-the `Catalyst hybrid compiler <https://github.com/pennylaneai/catalyst>`__.
-Catalyst allows you to compile the entire quantum-classical workflow,
-including any optimization loops, which allows for optimized performance, and
-the ability to run the entire workflow on accelerator devices as
-appropriate.
+hybrid just-in-time (JIT) compilation via the :func:`~.qjit` decorator and various
+hybrid compilers, which can be installed separately.
+
+The best supported and default compiler is the `Catalyst hybrid compiler
+<https://github.com/pennylaneai/catalyst>`__. Catalyst allows you to compile the entire
```

Comment:
I had a minor worry that saying "best supported and default" puts us in a `default.qubit`-like situation where everyone goes for this option because they don't have any motivation to use anything else. Wonder if it's worth rephrasing and just having "The Catalyst hybrid compiler allows you to ...".